### PR TITLE
Fixes #260: Added more fields to Compare Twitter Profiles App

### LINKED ISSUE #260: Add more fields to Compare Twitter Profiles App
- Issue type: Feature request 

### Short description
Add more fields for comparison in the app

### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #260 
Added more fields for comparison.

Demo link (if applicable): https://kavithaenair.github.io/compareTwitterProfiles/

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @singhpratyush @vibhcool @Achint08 Please review.

File: compareTwitterProfiles/index.html [hunk 1]
```diff
@@ -69,26 +69,51 @@
...
-                                    <h3 class="panel-title pull-left">User 2 Profile</h3>
+                                    <h3 class="panel-title pull-left">User 1 Profile</h3>
                                 </div>
                                 <div class="list-group">
                                     <div class="list-group-item">
                                         <img src="{{userData1.profile_image_url}}" alt="" style="pull-left">
-                                        <h4 class="list-group-item-heading">{{userData1.name}}</h4>
+                                        <h4 class="list-group-item-heading">{{userData.name}}</h4>
+                                    </div>
+                                    <div class="list-group-item"">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: extra closing inverted comma

File: compareTwitterProfiles/index.html [hunk 2]
```diff
@@ -69,26 +69,51 @@
...
+                                            <strong>{{userData.description}}</strong> Description
+                                        </h4>
                                     </div>
                                     <div class="list-group-item">
                                         <h4 class="list-group-item-heading">
-                                            <strong>{{userData1.followers_count}}</strong> Followers
+                                            <strong>{{userData.followers_count}}</strong> Followers
+                                        </h4>
+                                    </div>
+                                    <div class="list-group-item"">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: here too

File: compareTwitterProfiles/index.html [hunk 3]
```diff
@@ -69,26 +69,51 @@
...
+                                            <strong>{{userData.friends_count}}</strong> Following
                                         </h4>
                                     </div>
                                     <div class="list-group-item">
                                         <h4 class="list-group-item-heading">
-                                            <strong>{{userData1.statuses_count}}</strong> Statuses
+                                            <strong>{{userData.statuses_count}}</strong> Statuses
+                                        </h4>
+                                    </div>
+                                    <div class="list-group-item"">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: here too

File: compareTwitterProfiles/index.html [hunk 4]
```diff
@@ -69,26 +69,51 @@
...
+                                        <h4 class="list-group-item-heading">
+                                            <strong>{{userData.favourites_count}}</strong> Favourites
+                                        </h4>
+                                    </div>
+                                    <div class="list-group-item"">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: here too

File: compareTwitterProfiles/index.html [hunk 5]
```diff
@@ -69,26 +69,51 @@
...
+                                        <h4 class="list-group-item-heading">
+                                            <strong>{{userData.created_at}}</strong> Created On
+                                        </h4>
+                                    </div>
+                                    <div class="list-group-item"">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: here too

File: compareTwitterProfiles/index.html [hunk 6]
```diff
@@ -106,16 +131,41 @@ <h3 class="panel-title pull-left">User 2 Profile</h3>
...
                                         <h4 class="list-group-item-heading">
                                             <strong>{{userData1.statuses_count}}</strong> Statuses
                                         </h4>
                                     </div>
+                                    <div class="list-group-item"">
+                                        <h4 class="list-group-item-heading">
+                                            <strong>{{userData1.favourites_count}}</strong> Favourites
+                                        </h4>
+                                    </div>
+                                    <div class="list-group-item"">
```
Comment (djmgit) [inline compareTwitterProfiles/index.html]: here too

**Comment by djmgit** [comment]
@kavithaenair left some comments. Please rebase

**Comment by kavithaenair** [comment]
@djmgit @vibhcool Please re review.

**Pull request merged**